lighthouse-core/config/v2/default.json

```diff
@@ -0,0 +1,308 @@
+    "speed-index-metric": {},
+    "estimated-input-latency": {},
+    "time-to-interactive": {},
+    "user-timings": {},
+    "screenshots": {},
+    "critical-request-chains": {},
+    "manifest-exists": {},
+    "manifest-background-color": {},
+    "manifest-theme-color": {},
+    "manifest-icons-min-192": {},
+    "manifest-icons-min-144": {},
+    "manifest-name": {},
+    "manifest-short-name": {},
+    "manifest-short-name-length": {},
+    "manifest-start-url": {},
+    "theme-color-meta": {},
+    "content-width": {},
+    "deprecations": {},
+    "aria-allowed-attr": {
+      "path": "accessibility/aria-allowed-attr"
+    },
+    "aria-required-attr": {
+      "path": "accessibility/aria-required-attr"
+    },
+    "aria-valid-attr-value": {
+      "path": "accessibility/aria-valid-attr-value"
+    },
+    "aria-valid-attr": {
+      "path": "accessibility/aria-valid-attr"
+    },
+    "color-contrast": {
+      "path": "accessibility/color-contrast"
+    },
+    "image-alt": {
+      "path": "accessibility/image-alt"
+    },
+    "label": {
+      "path": "accessibility/label"
+    },
+    "tabindex": {
+      "path": "accessibility/tabindex"
+    },
+    "total-byte-weight": {
+      "path": "byte-efficiency/total-byte-weight"
+    },
+    "unused-css-rules": {
+      "path": "byte-efficiency/unused-css-rules"
+    },
+    "uses-optimized-images": {
+      "path": "byte-efficiency/uses-optimized-images"
+    },
+    "uses-responsive-images": {
+      "path": "byte-efficiency/uses-responsive-images"
+    },
+    "appcache-manifest": {
+      "path": "dobetterweb/appcache-manifest"
+    },
+    "dom-size": {
+      "path": "dobetterweb/dom-size"
+    },
+    "external-anchors-use-rel-noopener": {
+      "path": "dobetterweb/external-anchors-use-rel-noopener"
+    },
+    "geolocation-on-start": {
+      "path": "dobetterweb/geolocation-on-start"
+    },
+    "link-blocking-first-paint": {
+      "path": "dobetterweb/link-blocking-first-paint"
+    },
+    "no-console-time": {
+      "path": "dobetterweb/no-console-time"
+    },
+    "no-datenow": {
+      "path": "dobetterweb/no-datenow"
+    },
+    "no-document-write": {
+      "path": "dobetterweb/no-document-write"
+    },
+    "no-mutation-events": {
+      "path": "dobetterweb/no-mutation-events"
+    },
+    "no-old-flexbox": {
+      "path": "dobetterweb/no-old-flexbox"
+    },
+    "no-websql": {
+      "path": "dobetterweb/no-websql"
+    },
+    "notification-on-start": {
+      "path": "dobetterweb/notification-on-start"
+    },
+    "script-blocking-first-paint": {
+      "path": "dobetterweb/script-blocking-first-paint"
+    },
+    "uses-http2": {
+      "path": "dobetterweb/uses-http2"
+    },
+    "uses-passive-event-listeners": {
+      "path": "dobetterweb/uses-passive-event-listeners"
+    }
+  },
+
+  "report": {
+    "categories": {
+      "pwa": {
+        "name": "Progressive Web App",
+        "description": "",
+        "children": [
```

Comment:
`children` was used forseeing the whole `subcategory` discussion occurring and the likely scenario that we will need to add categories as children of other categories. Keeping it generic with children, IMO, leaves open the possibility without a breaking change. In the proposal, each child has a `type: "audit"|"category"|"some-newfangled-thing"` but since all we use are audits today it seemed unnecessary.